### DISCUSSION TITLE
Tasmota & Smart Meter EMH ehz-pw8

### DISCUSSION THREAD

**Discussion by FelixLammers0110** [arendst/Tasmota#18809] [Ideas]
Hello,
at what intervals are actually added more scripts for smart meters? I first had an EMH ehz-K that worked. Unfortunately, I had to change to the EMH ehz-pw8 meter due to a different measurement concept. Now I no longer receive the current consumption.
Greetings

**Comment by Jason2866** [comment]
Adding new meters is community driven. There is no update strategy.
If you add, your script/setup is welcome
